cmd/vicadmin/server.go

```diff
@@ -314,16 +323,15 @@ func (s *server) loginPage(res http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	// Render login page (shows up on non-POST requests):
-	sess, err := client(&rootConfig)
-	if err != nil {
-		log.Errorf("Could not render login page due to vSphere connection error: %s", err.Error())
-		http.Error(res, genericErrorMessage, http.StatusInternalServerError)
-		return
+	// Render login page (shows up on non-POST requests)
+	hostName, _ := os.Hostname()
```

Comment:
Thought about this some more. I'll change this so `hostName = "VCH"` when we get an error. It would be better to display "VCH Login" just "Login". Thanks @caglar10ur!